assets/js/components/GoogleChart/DateMarker.js

```diff
@@ -38,6 +39,12 @@ export default function DateMarker( { id, text } ) {
 		trackEvent( eventCategory, 'chart_line_view' );
 	}, [ eventCategory ] );
 
+	const trackTooltipOpen = useCallback( () => {
+		trackEvent( eventCategory, 'chart_tooltip_view' );
+	}, [ eventCategory ] );
+
+	const handleTooltipOpen = useThrottle( trackTooltipOpen, 5000 );
```

Comment:
This is great but not what I had in mind (my mistake!). I mixed up the difference between throttle and debounce which is what I think we want here instead and already have in the codebase. Basically, we want to let the first invocation through, and then ignore any subsequent invocations until it hasn't been called for 5s. As a throttle, it would still invoke the function every 5s, but no more often than that.

i.e.
```js
	const handleTooltipOpen = useDebounce( trackTooltipOpen, 5000, {
		leading: true,
		trailing: false,
	} );
```

I still think this hook is worth keeping as we should be using this instead of `throttle` directly inside components, but let's do that in a separate issue where we refactor with it at the same time.

Reply:
Changes made and seem to work well; I've created #7123 to migrate other usage of `throttle`/re-integrate this hook. 👍🏻